cts/cli/regression.tools.exp

```diff
@@ -8944,7 +8944,7 @@ Diff: +++ 0.1.0 (null)
 -- /cib/status/node_state[@id='1']
 -- /cib/status/node_state[@id='httpd-bundle-0']
 -- /cib/status/node_state[@id='httpd-bundle-1']
-+  /cib:  @validate-with=pacemaker-3.10, @num_updates=0, @admin_epoch=0
++  /cib:  @validate-with=pacemaker-X, @num_updates=0, @admin_epoch=0
```

Comment:
1. This doesn't cover the XML output in the next test.
2. I would prefer to strip out `validate-with` from the diff completely. The result will be different depending on whether `validate-with` actually changed or not. In other words, if `crm_mon.xml` had `validate-with` `pacemaker-3.10`, there would be no `@validate-with=pacemaker-X` in the diff.

Reply:
Thanks, overlooked the other one. Even though we'd still have to update the output the first time the version changes, I do prefer to verify that any given field is actually there.